api/src/main/resources/db/migration/V1.0.31__INSTITUTE_API.sql

```diff
@@ -0,0 +1,33 @@
+INSERT INTO SCHOOL_CATEGORY_CODE (SCHOOL_CATEGORY_CODE, LABEL, DESCRIPTION, LEGACY_CODE,
+                                  DISPLAY_ORDER, EFFECTIVE_DATE, EXPIRY_DATE, CREATE_USER,
+                                  CREATE_DATE, UPDATE_USER, UPDATE_DATE)
+VALUES ('SPEC_RPRT', 'Special Reporting', 'Special Reporting School', NULL, 10,
+        to_date('2020-01-01 00:00:00', 'YYYY-MM-DD HH24:MI:SS'),
+        to_date('2099-12-31 00:00:00', 'YYYY-MM-DD HH24:MI:SS'), 'IDIR/TRERICHA',
+        to_date('2019-12-20 00:00:00', 'YYYY-MM-DD HH24:MI:SS'), 'IDIR/TRERICHA',
+        to_date('2019-12-20 00:00:00', 'YYYY-MM-DD HH24:MI:SS'));
+
+UPDATE school
```

Comment:
@trev-dev We should have mentioned that we use uppercase for all scripts. The only exception should be values. Please update everything to uppercase (there's a hotkey for it in Intellij).